src/agent/application.cpp

```diff
@@ -376,4 +375,18 @@ void Application::SetBorderAgentOnInitState(void)
 }
 #endif
 
+#if OTBR_ENABLE_DBUS_SERVER
+DBus::DependentComponents Application::MakeDBusDependentComponents(void)
+{
+    return DBus::DependentComponents
+    {
+        mHost, *mPublisher
+#if OTBR_ENABLE_BORDER_AGENT
+            ,
+            mBorderAgent
+#endif
```

Comment:
nit: I think C++ 11 should already support trailing coma. Does it work if you put the comma after `*mPublisher`?